src/utilities/Error.js

```diff
@@ -13,16 +13,15 @@ class Error {
  */
   static handleErrorResponse(res, error) {
     if (error.errors) {
-      res.status(400).json({
+      return res.status(400).json({
         success: false,
         message: error.errors[0].message
       });
-    } else {
-      res.status(500).json({
```

Comment:
Same feedback as in 

https://github.com/andela/helios-ah-backend/pull/40/commits/f480a8dc1716d186663b82fb6a3d810fc48c9e93#r243206818

Please check